### DISCUSSION TITLE
Inline assistant messing up formating in shell script

### DISCUSSION THREAD

**Discussion by ruben-symbiose** [zed-industries/zed#21633] [Feature Requests]
I work on a bash script and keep running in to a problem where the inline assistant is messing up the formatting (indentation) as it runs through the script. It solves the task I give it, but I can't use the result due to the mess.

I suspect something is wrong with the formatting in the file in general, maybe tabs and spaces are used for indentation? Zed does occasionally mess up indentation and adds a space to little or to much, so something is definitely off.

I have tried finding a way to auto-format / clean up the script. but the options I have found, does nothing. Any pointers to what I can check to fix this?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
